stock_picking_limit_return_qty/wizard/stock_picking_return.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Cetmix OU
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ReturnPicking(models.TransientModel):
+    _inherit = "stock.return.picking"
+
+    @api.model
+    def _prepare_stock_return_picking_line_vals_from_move(self, stock_move):
+
+        res = super(
```

Comment:
@dessanhemrayev Could you check why wizard code is not accessed during tests (no coverage) ?

Reply:
Ok, wrong github rendering.